api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -1094,15 +1094,23 @@ def _mix_requested(kwargs, opt):
     def delay(self):
         return self._ctx.delay()
 
+    def _get_location_from_point_if_needed(self, location_or_point: Union[types.Location, types.Point]):
+        if isinstance(location_or_point, types.Location):
+            return location_or_point
+        elif isinstance(location_or_point, types.Point):
+            return types.Location(types.Point, "unnamed_point")
```

Comment:
```suggestion
            return types.Location(types.Point, None)
```
Need to make the location `None` to properly cache it